.travis.yml

```diff
@@ -6,30 +6,30 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
`--ignore-platform-reqs` should really only happen for `php: nightly`: all other versions should **not** have this flag, or else we will end up with incompatible dependencies.